src/settings/Settings.ts

```diff
@@ -623,10 +624,9 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         displayName: _td("IRC display name width"),
         default: 80,
     },
-    "useIRCLayout": {
-        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
-        displayName: _td("Enable experimental, compact IRC style layout"),
-        default: false,
+    "layout": {
+        supportedLevels: LEVELS_ROOM_SETTINGS_WITH_ROOM,
```

Comment:
Hmm, so it's per-room here...? The chatter in the bubbles room led me to think we were starting with an account level setting, just like it used to be. Maybe I missed something...?

Reply:
Does it hurt having the possibility to set it per room (for example via /-command or console)?
If it's planned to add a per room option later it wouldn't be in the way now and doesn't need to be added later.

Reply:
Hmm, I believe it's better to make smaller, iterative steps. It doesn't seem like we have resolved yet where it should be per-room, so let's leave this at account settings as it was before. If we get to point of agreeing on a per-room, we'll want to change much more than this line anyway.